### PR TITLE
Feat/uuid

### PR DESCRIPTION
**Pull request by TylorS**
This implements a simple library for generating cryptographically safe identifiers that works in both browsers and node environments.

### CODE REVIEW

File: packages/uuid/src/createUuid.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+import { generateRandomNumbers } from './generateRandomNumbers'
+import { generateUuid } from './generateUuid'
+import { pipe } from '@typed/functions'
+
+/**
+ * Creates a universally unique identifier that works in both browser and Node.js
+ * environments.
+ * @name createUuid(): Uuid
+ */
+export const createUuid = pipe(generateRandomNumbers, generateUuid) as () => Uuid
```
Comment (Frikki) [inline packages/uuid/src/createUuid.ts]: Suggestion:

`createUuid` -> `uuid`

I don’t see any need to have a factory name here, unless you plan to make other UUID versions.

File: packages/uuid/src/generateRandomNumbers.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
+const isBrowser = typeof process === 'undefined'
+
+const ARRAY_SIZE = 16
+
+declare const msCrypto: Crypto
+
+export const generateRandomNumbers: () => UuidArray = isBrowser
```
Comment (Frikki) [inline packages/uuid/src/generateRandomNumbers.ts]: Suggestion:

`generateRandomNumbers` -> `randomUuidSeed`

File: packages/uuid/src/generateUuid.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import { Uuid } from './types'
+import { UuidArray } from './generateRandomNumbers'
+
+export function generateUuid(randomNumbers: UuidArray): Uuid {
```
Comment (Frikki) [inline packages/uuid/src/generateUuid.ts]: Suggestion:

`generateUuid(randomNumbers: UuidArray)` -> `uuidV4(seed: UuidArray)`

**Comment by TylorS** [comment]
Thanks for the suggestions @frikki, I'll update the PR soon.

**Pull request merged**